Source: Gentoo, https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=610b59b8f9b471f8548a7b09162792e1acca5d31
Upstream: Unknown, Bugtracker is currently down
Reason: Fix build with ffmpeg 3.0

diff -Naur alsa-plugins-1.1.0/a52/pcm_a52.c alsa-plugins-1.1.0.new/a52/pcm_a52.c
--- alsa-plugins-1.1.0/a52/pcm_a52.c	2015-10-27 14:34:06.000000000 +0100
+++ alsa-plugins-1.1.0.new/a52/pcm_a52.c	2016-02-18 17:53:20.542070330 +0100
@@ -513,7 +513,7 @@
 		rec->inbuf = NULL;
 	}
 #if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(54, 28, 0)
-	avcodec_free_frame(&rec->frame);
+	av_frame_free(&rec->frame);
 #else
 	av_freep(&rec->frame);
 #endif
@@ -557,7 +557,7 @@
 {
 	struct a52_ctx *rec = io->private_data;
 #ifdef USE_AVCODEC_FRAME
-	rec->frame = avcodec_alloc_frame();
+	rec->frame = av_frame_alloc();
 	if (!rec->frame)
 		return -ENOMEM;
 	if (av_samples_alloc(rec->frame->data, rec->frame->linesize,
